qa/integration-tests/src/test/java/io/camunda/it/rdbms/db/fixtures/AuthorizationFixtures.java

```diff
@@ -24,15 +24,21 @@ private AuthorizationFixtures() {}
   public static AuthorizationDbModel createRandomized(
       final Function<Builder, Builder> builderFunction) {
     final var ownerId = nextStringId();
+
+    final var randomPermissionType1 = randomEnum(PermissionType.class);
+    var randomPermissionType2 = randomEnum(PermissionType.class);
+    while (randomPermissionType2 == randomPermissionType1) {
+      randomPermissionType2 = randomEnum(PermissionType.class);
+    }
```

Comment:
🔧 Just to make this more fancy and a bit optimized

```
final var randomPermissionType1 = randomEnum(PermissionType.class);
PermissionType randomPermissionType2;

do {
    randomPermissionType2 = randomEnum(PermissionType.class);
} while (randomPermissionType2 == randomPermissionType1);
```

Reply:
Why is this more optimized?

Reply:
If the first assignment is equal we will do 2 assignments. It doesn't matter which structure we use as far as I can see? I'm happy to change it but I don't see a different 😄 Maybe it's just too early for me 😛